src/MoBi.Core/Serialization/Converter/v10/Converter90To100.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Xml.Linq;
+using MoBi.Core.Domain.Model;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Formulas;
+using OSPSuite.Core.Serialization.Exchange;
+using OSPSuite.Utility.Extensions;
+using OSPSuite.Utility.Visitor;
+
+namespace MoBi.Core.Serialization.Converter.v10
+{
+   public class Converter90To100 : IMoBiObjectConverter,
+      IVisitor<IParameterStartValuesBuildingBlock>,
+      IVisitor<SimulationTransfer>,
+      IVisitor<IModelCoreSimulation>
+
+   {
+      private readonly OSPSuite.Core.Converters.v10.Converter90To100 _coreConverter;
+      private bool _converted;
+
+      public Converter90To100(OSPSuite.Core.Converters.v10.Converter90To100 coreConverter)
+      {
+         _coreConverter = coreConverter;
+      }
+
+      public bool IsSatisfiedBy(int version) => version == ProjectVersions.V9_0;
+
+      public (int convertedToVersion, bool conversionHappened) Convert(object objectToUpdate, IMoBiProject project)
+      {
+         _converted = false;
+         var (_, coreConversionHappened) = _coreConverter.Convert(objectToUpdate);
+         this.Visit(objectToUpdate);
+         return (ProjectVersions.V10_0, _converted || coreConversionHappened);
+      }
+
+      public (int convertedToVersion, bool conversionHappened) ConvertXml(XElement element, IMoBiProject project)
+      {
+         return _coreConverter.ConvertXml(element);
+      }
+
+      public void Visit(IParameterStartValuesBuildingBlock parameterStartValuesBuildingBlock)
+      {
+         //we need to update the formula of some predefined expressions parameters from PK-Sim v9 to ensure that a simulation can still be built
+         parameterStartValuesBuildingBlock?.FormulaCache.Each(convertFormula);
+      }
+
+      public void Visit(IModelCoreSimulation simulation)
+      {
+         Visit(simulation?.BuildConfiguration?.ParameterStartValues);
+      }
+
+      public void Visit(SimulationTransfer simulationTransfer)
+      {
+         Visit(simulationTransfer?.Simulation);
+      }
+
+      private void convertFormula(IFormula formula)
+      {
+         var explicitFormula = formula as ExplicitFormula;
+         if (explicitFormula == null)
+            return;
+
+         if (!explicitFormula.NameIsOneOf(
```

Comment:
@PavelBal @Yuri05 These were the only one that I could find in v9 with a division

Reply:
@PavelBal I'd love if you could check it out and let me know if anything was missed?